test/user.test.js

```diff
@@ -214,6 +215,98 @@ describe('User', function() {
       assert(u2.password === u1.password);
     });
 
+    it('invalidates the user\'s accessToken when the user is deleted By id', function(done) {
+      var usersId;
+      async.series([
+        function(next) {
+          User.create({ email: 'b@c.com', password: 'bar' }, function(err, user) {
+            usersId = user.id;
+            if (err) return done (err);
```

Comment:
Thanks. Fixed it. 
But but I kept the `function (err)` at the end of the series array still, because it won’t work without it